pkg/crc/cluster/cluster.go

```diff
@@ -92,6 +93,33 @@ func GetRootPartitionUsage(sshRunner *ssh.Runner) (int64, int64, error) {
 	return diskSize, diskUsage, nil
 }
 
+func UpdateSSHKeyToMachineConfig(ocConfig oc.Config, sshPublicKeyPath string) error {
```

Comment:
The pull secret method with a similar role is called `EnsurePullSecretPresentInTheCluster`, would be nice to have some consistency I think.